src/config/apiRoutes.ts

```diff
@@ -103,6 +103,7 @@ export const apiRoutes = {
 		invitations: '/invitations',
 		usersInvitation: '/users/invitations',
 		members: '/members',
+		addOrganization: '/orgs'
```

Comment:
We already have the 'root: '/orgs'' route; we can use it instead of creating a new one

Reply:
Resolved